src/config.ts

```diff
@@ -1376,6 +1382,47 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
 		features: ['stargate', 'ibc-transfer', 'no-legacy-stdTx', 'ibc-go'],
 		explorerUrlToTx: 'http://blockexplorer-1.testnet.saage.io/transactions/{txHash}',
 	},
+
+	{
+		rpc: 'https://tx-endpoint-test.imversed.com:443',
+		rest: 'https://query-endpoint-test.imversed.com:443',
+		chainId: 'imversed-test-1',
+		chainName: 'imversedtestnet',
+		stakeCurrency: {
+			coinDenom: 'IMV',
+			coinMinimalDenom: 'nimv',
+			coinDecimals: 6,
+			coinImageUrl: window.location.origin + '/public/assets/tokens/imversed.png',
+			coinGeckoId: 'imversedtest',
+		},
+		bip44: { coinType: 118 },
+		bech32Config: Bech32Address.defaultBech32Config('imv'),
```

Comment:
Please register the bech32 prefix imv here:
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
I make PR to register `imv` prefix https://github.com/satoshilabs/slips/pull/1306

Reply:
PR merged and prefix imv registered

Reply:
FYI imv is registered in the Mainnet column, not Testnet. I suppose that be addressed later